### PR TITLE
Update cross_agent_tests

### PR DESCRIPTION
**Pull request by kaylareopelle**
Changes were made October 5. The changes are all spelling corrections.

### CODE REVIEW

File: test/fixtures/cross_agent_tests/datastores/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Datastore instance tests
 
-The datastore instance tests provide attributes similar to what an agent could expect to find regarding a database configuration and specifies the expected [datastore instance metric](https://source.datanerd.us/agents/agent-specs/blob/master/Datastore-Metrics-PORTED.md#datastore-metric-namespace) that should be generated. The table below lists types attributes and whether they will always be included or optionally included in each test case.
+The datastore instance tests provide attributes similar to what an agent could expect to find regarding a database configuration and specifies the expected [datastore instance metric](https://source.datanerd.us/agents/agent-specs/blob/master/Datastore-Metrics-PORTED.md#datastore-metric-namespace) that should be generated. The table below lists types attributes and whether will will always be included or optionally included in each test case.
```
Comment (tannalynn) [inline test/fixtures/cross_agent_tests/datastores/README.md:3]: hmmm I'm not sure about this one, seems like they would be right?

Comment (fallwith) [reply test/fixtures/cross_agent_tests/datastores/README.md:3]: Yep. `will will` is certainly not right in any case

File: test/fixtures/cross_agent_tests/distributed_tracing/trace_context.json [hunk 2]
```diff
@@ -956,7 +956,7 @@
     "test_name": "tracestate_has_larger_version",
     "comment": [
       "If the new relic payload's version is higher than 0, with extra new fields, all ",
-      "the existing fields should be read and used, and the extra future fields should ",
+      "the existing fields should be read and used, and the extra future feilds should ",
```
Comment (fallwith) [inline test/fixtures/cross_agent_tests/distributed_tracing/trace_context.json:959]: `fields` was correct here and should not be changed

File: test/fixtures/cross_agent_tests/distributed_tracing/trace_context.json [hunk 3]
```diff
@@ -185,7 +185,7 @@
   {
     "test_name": "spans_disabled_in_parent",
     "comment": [ "If the parent is a New Relic agent with span events disabled it SHOULD omit span",
-      "id from the tracestate. This verifies agents propagate Trace Context payloads when the",
+      "id from the tracestate. This verifies agents propogate Trace Context payloads when the",
```
Comment (fallwith) [inline test/fixtures/cross_agent_tests/distributed_tracing/trace_context.json:188]: `propagate` is the correct spelling and should not be changed here.

**Comment by kaylareopelle** [comment]
Whoops, this reverted improvements previously introduced

**Pull request closed without merging**